blackrock_channel_data[] data
int32[] num_samples
string[] channel_labels
